cairo/tests/utils/strategies.py

```diff
@@ -377,6 +378,24 @@ def tuple_strategy(thing):
         ),
         _snapshots=st.just([]),
         created_accounts=st.just(set()),
+    ).map(
+        # Create the original state snapshot using copies of the tries
+        lambda state: State(
+            _main_trie=state._main_trie,
+            _storage_tries=state._storage_tries,
+            # Create deep copies of the tries for the snapshot,
+            # because otherwise mutating the main trie will also mutate the snapshot
+            _snapshots=[
+                (
+                    copy_trie(state._main_trie),
+                    {
+                        addr: copy_trie(trie)
+                        for addr, trie in state._storage_tries.items()
+                    },
+                )
+            ],
+            created_accounts=set(),
```

Comment:
created_accounts will always be empty 
Do we pass the condition at some point ?
```
    if (is_created == 0) {
        tempvar res = U256(new U256Struct(0, 0));
        return res;
    }
```

Reply:
as execution advances, this set will grow

Reply:
alternatively, @enitrat 's test can call `mark_account_created`

Reply:
modified the strategy.